src/Mvc/Mvc.Core/src/ApiConventionTypeAttribute.cs

```diff
@@ -83,6 +83,7 @@ private static bool IsAllowedAttribute(object attribute)
     {
         return attribute is ProducesResponseTypeAttribute ||
             attribute is ProducesDefaultResponseTypeAttribute ||
-            attribute is ApiConventionNameMatchAttribute;
+            attribute is ApiConventionNameMatchAttribute ||
+            attribute.GetType().FullName == "System.Runtime.CompilerServices.NullableContextAttribute";
```

Comment:
```suggestion
            attribute.GetType().FullName == "System.Runtime.CompilerServices.NullableContextAttribute";
```

is there a reason we can do `attribute is NullableContextAttribute` here?

Reply:
NullableContextAttribute is a compiler code spit into the users code, example from SharpLab:

https://sharplab.io/#v2:EYLgtghglgdgNAExAagD4AEAsACAYge3wAoAhCAJwH5tgBKAWACgBvJgXyafQCZszymrRh0ZA===